tests/moe/test_trellis_checkpoint_bridge.py

```
@@ -0,0 +1,344 @@
+
+        w13 = torch.stack(
+            [
+                _padded(assembled.gate_weights, True),
+                _padded(assembled.up_weights, True),
+            ],
+            dim=0,
+        )
+        w2 = _padded(assembled.down_weights, False)
+
+        def _rows(member_ids) -> torch.Tensor:
+            table = torch.ones((slots, 3 * local), dtype=torch.float16, device=device)
```

Ruff (0.16.2)

[warning] Function definition does not bind loop variable `slots`

(B023)